WRF/README.md

```diff
@@ -0,0 +1,32 @@
+# Exploring WRF data on AWS
+
+This example shows how to extract a timeseries of temperature values for some locations specified with lat/lon from the [dynamically downscaled climate data for Alaska](https://registry.opendata.aws/wrf-alaska-snap/).  This tutorial assumes you have basic knowledge about how to launch an EC2 instance on AWS, and basic knowledge about using the command line in Linux.
+
+Open the EC2 console, **switch to the US West (Oregon) region**, and choose "Launch instance".  Search for `ami-000c6d1a9c74d8bab` and select that image from the Community results.  Configure the instance if you wish (defaults are fine for this demo) and launch.
+
+Connect to the instance as the `centos` user.
+
+After logging in as the `centos` user, start by cloning this repository:
+
+```
+git clone https://github.com/ua-snap/tutorials
+cd tutorials/WRF
+```
+
+Next, **mount the WRF s3 data bucket**.  We use [s3fs](https://github.com/s3fs-fuse/s3fs-fuse) for this.
+
+```
+s3fs -o public_bucket=1,use_cache=/tmp,ro,allow_other,umask=0022 wrf-ak-ar5 ~/wrf-ak-ar5
+```
+
+Finally, run the script inside the `pipenv` environment:
+
+```
+pipenv run python extract_points_wrf.py
+```
+
+After a few minutes, the script should produce a file named `extracted.csv`.
+
+## What next?
+
+ * Modify the script to pull data for a community you are interested in.
```

Comment:
Any other ideas on how to modify or extend this simple script?  Or would those be better as other tutorials?

Reply:
I think this one being simple and straightforward is a good place to start.  Maybe this will be a 'beginners' tutorial, and we can do more 'advanced' ones in the future? I am also not sure how many of these we _really_ need to make.  This one covers some good bases, like how to get to the data in an AMI, and how to extract a point of interest.  

That said, we could make it possible to put in a user defined lat/lon point in EPSG:3338 and it will return data if it is in the domain and an error if it is not.  Could be a CLI argument, or we could turn this into something folks can mess with in the interpreter? 

I wonder if this could fit into a Jupyter notebook or something like that, so people could have runnable snippets describing the steps?  I am not sure how that would be done in the context of an AMI, but is another approach. It is also more time spent, which at this stage in the game might not be best. 

All in all, I like this as the tutorial we link to from the AWS Open Data Program page.